zproject/prod_settings_template.py

```diff
@@ -1,7 +1,7 @@
 from typing import Any, Dict, Tuple
 
 ################################################################
-# Zulip Server settings.
+# Zulip Server settings. (0)
```

Comment:
I don't think we intend to move add these markers in master, as they aren't relevant to users and may become out of date; I think Alex mentioned this idea mostly to help with our discussion of how to organize these.